### PR TITLE
Add options to list private and published posts.

### PR DESCRIPTION
**Pull request by punchagan**
For the sake of completeness, `nikola status` should also be able to list private and published posts.

### CODE REVIEW

File: nikola/plugins/command/status.py [hunk 1]
```diff
@@ -123,6 +147,12 @@ def _execute(self, options, args):
         if options['list_drafts']:
             for post in posts_drafts:
                 print("Draft: '{0}' ({1}; source: {2})".format(post.meta('title'), post.permalink(), post.source_path))
+        if options['list_private']:
+            for post in posts_private:
+                print("Private: '{0}' ({1}; source: {2})".format(post.meta('title'), post.permalink(), post.source_path))
+        if options['list_published']:
+            for post in posts_published:
+                print("Published: '{0}' ({1}; source: {2})".format(post.meta('title'), post.permalink(), post.source_path))
         print("{0} posts in total, {1} scheduled, and {2} drafts.".format(posts_count, len(posts_scheduled), len(posts_drafts)))
```
Comment (Kwpolska) [inline nikola/plugins/command/status.py]: add it to the count there

**Comment by punchagan** [comment]
Done.

**Comment by Kwpolska** [comment]
Thanks!

(I assume you did all the necessary testing.)

**Comment by punchagan** [comment]
I have.

**Pull request merged**